main.tf

```diff
@@ -47,13 +47,27 @@ module "ecs_fargate" {
   container_cpu                = 4096
   container_memory             = 8192
   container_memory_reservation = 4096
+  enable_autoscaling           = false
+
   lb_http_ports = {
+  #  default = {
+  #    listener_port     = 80
+  #    target_group_port = 9000
+  #  }
+  }
+
+  lb_https_ports = {
```

Comment:
Is it possible to have this configuration with SSL as default but allow module users to disable and use http?